common/script/fns/cron.js

```diff
@@ -36,6 +36,9 @@ module.exports = function(user, options) {
   }
   user.auth.timestamps.loggedin = new Date();
   user.lastCron = now;
+  if (_.isFinite(+user.preferences.timezoneOffset)) {
```

Comment:
But why `_.isFinite`? Wouldn't `_.isNumber` make more sense? Checking against the number not being infinity is a little confusing to me. 

Also, is there any case in which the value is set, but it is _not_ a number? I'd guess that the two value types are undefined or Number, right? In which case, I think `if (timezoneOffset in user.preferences)` would do just as well.